src/components/views/rooms/EditMessageComposer.js

```diff
@@ -176,16 +180,18 @@ export default class EditMessageComposer extends React.Component {
         const editedEvent = this.props.editState.getEvent();
         const editContent = createEditContent(this.model, editedEvent);
         const newContent = editContent["m.new_content"];
-        if (!this._isModifiedOrSameAsOld(newContent)) {
-            return;
+
+        // If content is modified then send an updated event into the room
+        if (this._isContentModified(newContent)) {
+            const roomId = editedEvent.getRoomId();
+            this._cancelPreviousPendingEdit();
+            this.context.matrixClient.sendMessage(roomId, editContent);
         }
-        const roomId = editedEvent.getRoomId();
-        this._cancelPreviousPendingEdit();
-        this.context.matrixClient.sendMessage(roomId, editContent);
 
+        // close the event editing and focus composer
         dis.dispatch({action: "edit_event", event: null});
```

Comment:
This will now gets run when the message hasn't been modified, or at least it would but the click handler won't fire because the button is disabled. Did you just remove the early return on purpose or is the code below actually meant to run?

Reply:
This is for if you change the message from `a` to `ab` then back to `a` so no edit is actually made bu the Save button will be enabled